radixdlt-core/radixdlt/src/main/java/com/radixdlt/client/ArchiveApiModule.java

```diff
@@ -158,7 +160,18 @@ public JsonRpcHandler networkTransactionDemand(HighLevelApiHandler highLevelApiH
 	}
 
 	@ProvidesIntoSet
-	public EventProcessorOnRunner<?> clientApiStore(ClientApiStore clientApiStore) {
+	private EventProcessorOnRunner<?> newBatch(
+			BerkeleyClientApiStore berkeleyClientApiStore
+	) {
+		return new EventProcessorOnRunner<>(
+				Runners.APPLICATION,
+				AtomsCommittedToLedger.class,
+				berkeleyClientApiStore.newBatchEventProcessor()
```

Comment:
we usually use the format {eventName}eventProcessor.
might be better to rename it to `atomsCommittedToLedgerEventProcessor` (also applies to this method)